src/Riok.Mapperly/Riok.Mapperly.Roslyn4.0.props

```diff
@@ -1,5 +1,9 @@
 <Project>
 
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);ROSLYN3_9_OR_GREATER;ROSLYN3_10_OR_GREATER;ROSLYN4_0_OR_GREATER</DefineConstants>
```

Comment:
Here we define a `ROSLYN3_10_OR_GREATER`, however, Mapperly never references an API version of `3.10` of Roslyn. Therefore I would not define this constant. Instead, I think it would make sense to define constants without the `_OR_GREATER` suffix (eg. `ROSLYN_4_0`), this would allow to simplify some conditions.